moderncvbodyi.sty

```diff
@@ -97,8 +97,8 @@
     \hspace*{\separatorcolumnwidth}%
     \begin{minipage}[t]{\doubleitemcolumnwidth}#5\end{minipage}}}
 
-\renewcommand*{\cvlistitem}[2][.25em]{%
-  \cvitem[#1]{}{\listitemsymbol\begin{minipage}[t]{\listitemcolumnwidth}#2\end{minipage}}}
+\renewcommandx*{\cvlistitem}[3][1=.25em, usedefault, 2=0em]{%
+  \cvitem[#1]{}{\listitemsymbol\hspace{#2}\begin{minipage}[t]{\listitemcolumnwidth - #2}#3\end{minipage}}}
 
 \renewcommand*{\cvlistdoubleitem}[3][.25em]{%
```

Comment:
I think that additional space between `\listitemsymbol` and the text should also be applied for `\cvlistdoubleitem{...}`.

Reply:
new implementation with option 2